model/src/main/proto/profile.proto

```diff
@@ -78,6 +78,9 @@ message Profile {
 
   // Determines if the in-exploration continue button animation has been seen by the user or not.
   bool is_continue_button_animation_seen = 15;
+
+  // Represents user selected content-language.
+  OppiaLanguage oppia_language = 16;
```

Comment:
Why do we need this if we already store per-profile language selections through TranslationController?

Reply:
This is still being used to by AppLanguageSelection which is stored per profile with the user's language selection.

Reply:
I'm not sure that I understand--AppLanguageSelection is a proto that doesn't interact with profile protos at all. Could you maybe clarify further on why this field is actually needed?

Reply:
Ooh!! i had endless trouble trying to combine data from TranslationController and ProfileManagementController while displaying the current user options that's why i fell back to storing OppiaLanguage in the profile too.